### PR TITLE
Document 39 hooks for WP-Parser

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/30184

### CODE REVIEW

File: admin-views/tribe-options-display.php [hunk 1]
```diff
@@ -10,6 +10,18 @@
 	$template_options[$templates[$template]] = $template;
 }
 
+/**
+ * Filter the array of views that are registered for the tribe bar
+ * @param array array() {
+ *     Associative array of views
+ *
```
Comment (barryhughes) [inline admin-views/tribe-options-display.php]: Wondering, should it be made clearer that this is an array of arrays (only the inner arrays are associative)?

Comment (zbtirrell) [inline admin-views/tribe-options-display.php]: "Array of Associative array views"?

Comment (barryhughes) [inline admin-views/tribe-options-display.php]: "Array of views, where each view is itself represented by an associative array consisting of these keys:"

...too wordy?

**Comment by borkweb** [comment]
:+1:
